### ISSUE TITLE
cdktf deploy panics

### ISSUE THREAD

**Issue by Zubnix** [hashicorp/terraform-cdk#688]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

Typescript
cdktf 0.3.0

### Affected Resource(s)

provider-google
ContainerCluster

### Debug Output

```
import { ContainerCluster } from "@cdktf/provider-google";
import { App, TerraformStack } from "cdktf";
import { Construct } from "constructs";

const app = new App()

class GCloudStack extends TerraformStack {
  constructor (scope: Construct) {
    super(scope, `stack`);
    new ContainerCluster(this, `cluster`, { name: `cluster` })
  }
}

new GCloudStack(app)

app.synth()
```
gives
https://gist.github.com/Zubnix/f6f0a3dfabb9aab32d99ac070ccd2d5d

### Expected Behavior

Asking for deploy confirmation

### Actual Behavior

panic

### Steps to Reproduce

Run above sample

### Important Factoids

Somehow this initially worked using a more complex setup, a series of deploys & destroys suddenly gave this state. The only thing that does seem to work is to remove the cluster resource...

I tried removing all stack state I could see, destroy all resources in an attempt to reset the stack somehow...

However if I manually navigate to the cdktf.out directory and manually run `terraform apply` everything is ok.... 🤷🏻

**Comment by DanielMSchmidt** [comment]
Based on the stacktrace `terraform show -json <planfile>` fails ([invoked here](https://github.com/hashicorp/terraform-cdk/blob/febfc4110bfd67a57fc51d22c85e6a0bb91de682/packages/cdktf-cli/bin/cmds/ui/models/terraform-cli.ts#L52). Which version of terraform are you using?

**Comment by DanielMSchmidt** [comment]
I tried to create a minimal example, letting it apply and destroy 100 times now.
https://github.com/DanielMSchmidt/cdktf-issue-688

**Comment by DanielMSchmidt** [comment]
It could be this bug in terraform: https://github.com/hashicorp/terraform/pull/28624
The bug got fixed and released, terraform 0.15.3 has the fix, please try updating terraform 👍

**Comment by Zubnix** [comment]
I switched to vanilla terraform in the meanwhile. Tried to reproduce it just now and everything seems to be working.
